nx-X11/programs/Xserver/hw/nxagent/Window.c

```diff
@@ -1003,6 +1008,13 @@ void nxagentSwitchAllScreens(ScreenPtr pScreen, Bool switchOn)
       }
     }
 
+    /*
+     * FIXME: These are 0 most of the time nowadays. The effect is,
+     * that the window is moving a bit to right/bottom everytime
+     * Fullscreen is left. To fix this ask the window manager the
```

Comment:
s/ask the window manager the frame extents/query the frame extents from the window manager/